nemo_curator/modules/config.py

```diff
@@ -84,9 +84,15 @@ def __post_init__(self):
             raise ValueError(
                 "Finding fuzzy duplicates requires a cache directory accessible via all workers to store intermediates"
             )
-        if not self.false_positive_check:
-            raise NotImplementedError(
-                "Skipping false positive checks is not supported at the moment"
+        if self.false_positive_check:
+            warnings.warn(
+                "Identifying false positives during the Minhash deduplication is computationally expensive."
+                " For improved performance consider setting this to False"
+            )
+        if not self.false_positive_check and self.char_ngrams < 20:
+            warnings.warn(
+                "Using a small char_ngrams value might lead to a large number of false positives during deduplication."
```

Comment:
It could be useful adding the exact false positive rate we observed. Also good to add to the docs, but that can come later since I think an overhaul of the fuzzy dedup docs is already on the plan.

Reply:
Just added a (~5%) in the warning to give users an estimate. The docs overhaul can contain more info on the actual false positive rates.